lib/browser.ts

```diff
@@ -851,11 +851,11 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    * browser.get('https://angularjs.org/');
    * expect(browser.getCurrentUrl()).toBe('https://angularjs.org/');
    *
-   * @param {string} destination Destination URL.
+   * @param {string=} opt_destination Destination URL. Uses 'config.baseUrl' by default
```

Comment:
Thanks!
I see typings in comments everywhere in this file and project. Should this be fixed globally? 

I am not sure how to better write this line, could you help?